capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx

```diff
@@ -67,7 +64,7 @@ export default function ExperienceInfo({ onUpdateRestaurants }) {
     if (!isRated) {
       setNewReview(resetRatingForm);
     }
-  }, [currentRestaurant, setNewReview]);
+  }, [currentRestaurant, setNewReview, isRated]);
```

Comment:
thanks for fixing this!

Reply:
Thanks for bringing that up to my attention!